### ISSUE TITLE
Copy-DbaAgentAlert bug

### ISSUE THREAD

**Issue by potatoqualitee** [sqlcollaborative/dbatools#1542]
## Is this:
_Just place "x" to indicate selection (e.g. "[x]")_

- [ ] *feature* 
- [x] *bug*:

## Details

````
WARNING: [Copy-DbaAgentAlert][22:20:31] Alert [Replication: Subscription reinitialized after validation failure] has job [] configured as repsonse. The job does not exist on
destination . Skipping.
SourceServer      : sql2014
DestinationServer : sql2016
Name              : Replication: Subscription reinitialized after validation failure
Type              : Agent Alert
Status            : Skipped
DateTime          : 2017-07-02 22:20:31.725

WARNING: [Copy-DbaAgentAlert][22:20:31] Alert [Severity 016] has job [] configured as repsonse. The job does not exist on destination . Skipping.
SourceServer      : sql2014
DestinationServer : sql2016
Name              : Severity 016
Type              : Agent Alert
Status            : Skipped
DateTime          : 2017-07-02 22:20:31.747

WARNING: [Copy-DbaAgentAlert][22:20:31] Alert [Severity 017] has job [] configured as repsonse. The job does not exist on destination . Skipping.
SourceServer      : sql2014
DestinationServer : sql2016
Name              : Severity 017
Type              : Agent Alert
Status            : Skipped
DateTime          : 2017-07-02 22:20:31.764

WARNING: [Copy-DbaAgentAlert][22:20:31] Alert [Severity 018] has job [] configured as repsonse. The job does not exist on destination . Skipping.
SourceServer      : sql2014
````

**Comment by wsmelton** [comment]
This is part of the fix in #1276  guess it needs adjusting.

**Comment by potatoqualitee** [comment]
fixed actually!
